packages/rest/src/rest.server.ts

```diff
@@ -167,6 +167,12 @@ export class RestServer extends Context implements Server, HttpServerLike {
     this._setupRequestHandler(options);
 
     this.bind(RestBindings.HANDLER).toDynamicValue(() => this.httpHandler);
+
+    this._httpServer = new HttpServer(this.requestHandler, {
+      port: options.port,
+      host: options.host,
+      protocol: options.protocol || 'http',
+    });
```

Comment:
Please move the initialization of `_httpServer` to `_setupRequestHandler` method below.